index.js

```diff
@@ -252,7 +252,8 @@ class K8sVMExecutor extends Executor {
         const podConfig = yaml.safeLoad(podTemplate);
 
         const diskConfig = annotations[DISK_RESOURCE] || '';
-        const nodeSelectors = diskConfig.toUpperCase() === 'HIGH' ? { disk: 'high' } : {};
+        const nodeSelectors = diskConfig.toUpperCase() === 'HIGH' ?
+            { 'screwdriver.cd/disk': 'high' } : {};
```

Comment:
Shall we make this label configurable? Otherwise later on if we decided to change the label, we need to actually change this line.

Reply:
I see, so something like `options -> kubernetes -> labels -> disk`, then bind it it to the executor as part of the constructer?

Reply:
Added an extra commit to configure the disk label.  Should not be blocked on the queue-worker implementation since we default to `screwdriver.cd/disk` if the option is missing.